Dockerfile

```diff
@@ -32,10 +31,44 @@ RUN apt-get install -y --no-install-recommends --no-install-suggests wget && \
     tar zxf ceres-solver-2.1.0.tar.gz && \
     mkdir ceres-build && \
     cd ceres-build && \
-    cmake ../ceres-solver-2.1.0 -GNinja && \
+    cmake ../ceres-solver-2.1.0 -GNinja \
+        -DCMAKE_INSTALL_PREFIX=/ceres_installed && \
     ninja install
+RUN cp -r /ceres_installed/* /usr/local/
 
 # Build pyceres.
 COPY . /pyceres
 WORKDIR /pyceres
-RUN pip install . -vv
+RUN pip install --upgrade pip
+RUN pip wheel . --no-deps -w dist-wheel -vv && \
+    whl_path=$(find dist-wheel/ -name "*.whl") && \
+    echo $whl_path >dist-wheel/whl_path.txt
+
+
+#
+# Runtime stage.
+#
+FROM ubuntu:${UBUNTU_VERSION} as runtime
+
+# Install minimal runtime dependencies.
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends --no-install-suggests \
+        libgoogle-glog0v5 \
+        libspqr2 \
+        libcxsparse3 \
+        libatlas3-base \
+        python-is-python3 \
+        python3-minimal \
+        python3-pip
+
+# Copy installed library in the builder stage.
+COPY --from=builder /ceres_installed/ /usr/local/
+
+# Install pyceres.
+COPY --from=builder /pyceres/dist-wheel /tmp/dist-wheel
+RUN pip install --upgrade pip
+RUN cd /tmp && whl_path=$(cat dist-wheel/whl_path.txt) && pip install $whl_path
+RUN rm -rfv /tmp/*
+
+# # Verify if pyceres library is accessible from python.
+RUN python -c "import pyceres"
```

Comment:
I wonder if it would make sense to include the source code here in the runtime image as well. I mean:
```
COPY . /pyceres
WORKDIR /pyceres
```

My initial thought was that this runtime image could be used as a base image in another Dockerfile, , so I omitted the code.

FYI, I ran the `test_python_cost.py` file and it worked fine in the runtime images.

Reply:
I have no idea how such Docker image is supposed to be used, so I can't really advise on this.

Reply:
yes, I will keep as it is now.